Assets/Plugins/Source/Editor/EditorWindows/EOSPluginSettingsWindow.cs

```diff
@@ -104,10 +105,7 @@ protected override void RenderWindow()
             {
                 foreach (var configurationSectionEditor in configEditors)
                 {
-                    GUILayout.Label(configurationSectionEditor.GetLabelText(), EditorStyles.boldLabel);
-                    GUIEditorUtility.HorizontalLine(Color.white);
-                    configurationSectionEditor.Render();
-                    EditorGUILayout.Space();
+                    configurationSectionEditor.RenderAsync();
```

Comment:
This has a signature of `async Task`. Should this be `await`'d?

Reply:
Interestingly, this is not seen as an issue because the list of configEditors implements the `IConfigEditor` interface, and it is not possible to specify that the implementation of that interface should be async, it just specifies that it returns a task. To ameliorate the situation, I used the discard pattern to avoid that - even though it doesn't cause problems during runtime, and doesn't throw warnings in the editor or during compilation.